### PR TITLE
Clean next

### PR DESCRIPTION
**Pull request by nouman-novatore**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -24,6 +26,7 @@
     namespace :v1 do
 
       resources :ratings
+      # get 'notification_check' => 'notifications#notification_check'
```
Comment (xishoon) [inline config/routes.rb]: Remove all unnecessary commented code

File: app/controllers/api/v1/orders_controller.rb [hunk 2]
```diff
@@ -64,8 +64,35 @@ def create
...
+        elsif @order.status == 'dropped_to_vendor'
+          @title="Dropped to Vendor !"
+          @message="Rider has dropped to vendor"
+        elsif @order.status == 'picked_up_from_vendor'
+          @title="Be ready to collect !"
+          @message="Your order has been picked up from Shop"
+        elsif @order.status == 'completed'
+          @title="Order completed !"
+          @message="Dont forget to rate and comments"
+        end
```
Comment (xishoon) [inline app/controllers/api/v1/orders_controller.rb]: Use switch statement instead of If else

File: db/schema.rb [hunk 3]
```diff
@@ -117,17 +134,17 @@
...
   create_table "shop_capabilities", force: :cascade do |t|
     t.bigint "capability_id"
-    t.bigint "shop_id"
-    t.integer "laundry_id"
+    t.bigint "shops_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["capability_id"], name: "index_shop_capabilities_on_capability_id"
-    t.index ["shop_id"], name: "index_shop_capabilities_on_shop_id"
+    t.index ["shops_id"], name: "index_shop_capabilities_on_shops_id"
```
Comment (nouman-novatore) [inline db/schema.rb]: ok

File: app/controllers/api/v1/orders_controller.rb (hunk 2 above)
Comment (nouman-novatore) [reply app/controllers/api/v1/orders_controller.rb]: done

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -30,4 +30,6 @@ def verify_user_logged_in?
 		}, status: 200 and return unless user_signed_in?
 	end
 
+	
+
 end
```
[the remark this answers is not in the document]
Comment (nouman-novatore) [reply app/controllers/application_controller.rb]: it doesn't matter.

File: app/models/order.rb [hunk 5]
```diff
@@ -1,25 +1,40 @@
...
+  validates :cost,  presence: true
+  validates :delivery_date,  presence: true
+  validates :delivery_time,  presence: true
+  validates :order_type,  presence: true
+  validates :shop_id,  presence: true
+  validates :pick_lat,  presence: true
+  validates :pick_lng,  presence: true
+  validates :shop_lat,  presence: true
+  validates :shop_lng,  presence: true
+
```
[the remark this answers is not in the document]
Comment (nouman-novatore) [reply app/models/order.rb]: done

File: config/routes.rb (hunk 1 above)
Comment (nouman-novatore) [reply config/routes.rb]: done

File: config/database.yml [hunk 6]
```diff
@@ -1,8 +1,8 @@
 development:
   host: localhost
   username: postgres
-  password: postgres
-  database: clean_bee_dev_db
+  password: CleanBee
+  database: clean_b_rider
   adapter: postgresql
   encoding: unicode
   pool: 5
```
[the remark this answers is not in the document]
Comment (nouman-novatore) [reply config/database.yml]: done

File: Gemfile.lock [hunk 7]
```diff
@@ -336,7 +343,7 @@ DEPENDENCIES
   web-console (>= 3.3.0)
 
 RUBY VERSION
-   ruby 2.5.3p105
+   ruby 2.5.1p57
```
[the remark this answers is not in the document]
Comment (nouman-novatore) [reply Gemfile.lock]: done

**Pull request merged**